sample_project/Assets/SampleViewer/Samples/OverviewMap/Scripts/LocationMarker.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using Esri.ArcGISMapsSDK.Components;
+using Esri.ArcGISMapsSDK.Samples.Components;
+using Esri.ArcGISMapsSDK.Utils.GeoCoord;
+using Esri.GameEngine.Geometry;
+using UnityEngine;
+
+public class LocationMarker : MonoBehaviour
+{
+    [SerializeField] private ArcGISCameraComponent cameraComponent;
+
+    private ArcGISCameraControllerComponent cameraController;
+    private ArcGISLocationComponent locationComponent;
+    private float northOffset = 225.0f;
+
+    private void Awake()
+    {
+        cameraController = FindFirstObjectByType<ArcGISCameraControllerComponent>();
+        locationComponent = GetComponent<ArcGISLocationComponent>();
+    }
+
+    private void Update()
+    {
+        cameraComponent.GetComponent<ArcGISLocationComponent>().Position = new ArcGISPoint(
+            cameraController.GetComponent<ArcGISLocationComponent>().Position.X,
+            cameraController.GetComponent<ArcGISLocationComponent>().Position.Y,
+            cameraComponent.GetComponent<ArcGISLocationComponent>().Position.Z, ArcGISSpatialReference.WGS84());
+        locationComponent.Position = new ArcGISPoint(
+            cameraController.GetComponent<ArcGISLocationComponent>().Position.X,
+            cameraController.GetComponent<ArcGISLocationComponent>().Position.Y, locationComponent.Position.Z,
+            ArcGISSpatialReference.WGS84());
```

Comment:
can you retrieve `cameraController.GetComponent<ArcGISLocationComponent>().Position.X` only once?

Reply:
It would need to be stored as a variable and I don't think that is necessary since it is only used here. To shorten the line `cameraController.GetComponent<ArcGISLocationComponent>()` could be stored as a variable, but there's not need to specifically store the X value.

Reply:
`cameraController.GetComponent<ArcGISLocationComponent>().Position.X` and `cameraController.GetComponent<ArcGISLocationComponent>().Position.Y`  are retrieved twice respectively and can be stored as variables.